cro-clib/src/types.rs

```diff
@@ -34,6 +42,16 @@ pub struct CroAddress {
 }
 
 #[repr(C)]
+#[derive(Clone)]
+pub struct CroStakedState {
+    pub nonce: u64,
```

Comment:
what about the address, jailed, ...? but I guess this may be replaced/redesigned later

Reply:
yes, i'll add more state.